### PR TITLE
LIB-1338 fix(badge): update height

### PR DESCRIPTION
**Pull request by Trebh**
Jira: [LIB-1338](https://fiscozen.atlassian.net/browse/LIB-1338)

### CODE REVIEW

**Review by panicofr — APPROVED** [review]

File: packages/badge/src/FzBadge.vue [hunk 1]
```diff
@@ -39,8 +39,8 @@ const mapColorToClasses = {
 };
 
 const mapSizeToClasses = {
-  sm: "text-xs px-8 h-16 w-16",
-  md: "text-sm px-12 h-20 w-20",
+  sm: "text-xs px-8 h-20 w-16",
```
Comment (Crish15) [inline packages/badge/src/FzBadge.vue:42]:
```suggestion
  sm: "text-xs px-8 h-20 w-20",
```

File: packages/badge/src/FzBadge.vue [hunk 2]
```diff
@@ -39,8 +39,8 @@ const mapColorToClasses = {
...
+  md: "text-sm px-12 h-24 w-20",
```
Comment (Crish15) [inline packages/badge/src/FzBadge.vue:43]:
```suggestion
  md: "text-sm px-12 h-24 w-24",
```

**Review by Crish15 — CHANGES_REQUESTED** [review]
also the width must be changed

**Review by oxcened — COMMENTED** [review]
looks like you didn't update the snapshots

File: packages/badge/src/FzBadge.vue (hunk 2 above)
Comment (oxcened) [reply packages/badge/src/FzBadge.vue:43]:
```suggestion
  md: "text-sm px-12 size-24",
```
or just

File: packages/badge/src/FzBadge.vue (hunk 1 above)
Comment (oxcened) [reply packages/badge/src/FzBadge.vue:42]:
```suggestion
  sm: "text-xs px-8 size-20",
```

**Review by oxcened — APPROVED** [review]

**Pull request merged**